src/kernels/raw/finder/localKernelFinder.web.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+
+import type * as nbformat from '@jupyterlab/nbformat';
+import { injectable } from 'inversify';
+import { CancellationToken } from 'vscode';
+import { LocalKernelConnectionMetadata } from '../../../kernels/types';
+import { traceDecoratorVerbose, ignoreLogging, traceDecoratorError } from '../../../platform/logging';
+import { Resource } from '../../../platform/common/types';
+import { captureTelemetry } from '../../../telemetry';
+import { Telemetry } from '../../../webviews/webview-side/common/constants';
+import { ILocalKernelFinder } from '../types';
+import { TraceOptions } from '../../../platform/logging/types';
+
+// The plan is to remove this later and refactor the NotebookControllerManager to not know the difference between local and remote
+// Instead there will be a IKernelFinder for web and an IKernelFinder for node. It will coalesce the local/remote bits.
+// But for now, it was simpler to just make this kernel finder that returns nothing.
+@injectable()
+export class LocalKernelFinder implements ILocalKernelFinder {
```

Comment:
This class should be temporary. The reason this exists is because the NotebookControllerManager is doing special logic for local. 

I hope to remove this entirely in a IKernelFinder implementation later.